compysition/actors/zeromq.py

```diff
@@ -96,7 +96,7 @@ def __init__(self, name, port=DEFAULT_PORT, transmission_protocol=TCP, socket_fi
                                     self.IPC: "ipc://{0}".format(socket_file),
                                     self.INPROC: "inproc://{0}".format(socket_file)}
 
-        if transmission_protocol in self.format_connection.keys():
+        if self.format_connection.get(transmission_protocol, None):
```

Comment:
Why was this changed? I think ideally this would be `if transmission_protocol in self.format_connection`, we can drop the `.keys()`. This also changes what we're doing (not in practice, it still works fine obviously), as instead of checking if the key is in the dict, we're evaluating the value of that key.

Reply:
My initial goal was simply to remove the `.keys()`.  When I made this change I was thinking the transmission_protocol value should never be None but after further review of the context it looks like it never will be.  So, I agree `if transmission_protocol in self.format_connection` is probably more efficient.